### PR TITLE
Data frame

### PR DESCRIPTION
**Pull request by jordansread**
for #322

### CODE REVIEW

**Comment by jordansread** [comment]
whoops, looks like there is something in the examples that isn't in tests.

**Comment by ldecicco-USGS** [comment]
Is this ready to merge?

**Comment by jordansread** [comment]
I think so..?

**Comment by lindsayplatt** [comment]
@jread-usgs looks like this broke axis labeling, but only seems to be for side 2 and 4:

``` r
gs <- gsplot() %>% points(1,2)
gs
```

image

**Comment by jordansread** [comment]
@lindsaycarr we should revert this then, right?

**Comment by lindsayplatt** [comment]
I'd say yes...how exactly do we do that?

File: R/function_args.R [hunk 1]
```diff
@@ -14,18 +14,15 @@ function_args <- function(package, name, object, ..., use.default=paste0(name,'.
   params <- list(...)
   
   if (!missing(object)) {
-    if (!is.null(names(object)))
-      params <- append(object, params)
-    else {
-      params <- append(list(object), params)
-    }
+    params <- append_params(object, params)
```
Comment (lindsayplatt) [inline R/function_args.R:17]: just isolated this as the issue that caused the labels to disappear.

Comment (jordansread) [reply R/function_args.R:17]: ahhh

File: R/axis.R [hunk 2]
```diff
@@ -63,8 +63,7 @@ axis <- function(object, ...) {
   override("graphics", "axis", object, ...)
 }
 
-#' @importFrom utils head
-axis.gsplot <- function(object, ..., n.minor=0, tcl.minor=0.15, reverse=NULL, append=FALSE) {
+axis.gsplot <- function(object, ..., n.minor=0, tcl.minor=0.15, reverse=NULL) {
```
Comment (lindsayplatt) [inline R/axis.R:66]: @jread-usgs why did we delete append=FALSE?

**Comment by jordansread** [comment]
I don't know....Maybe that happened accidentally w a merge? Let's track down the commit and figure out why

**Pull request merged**